### ISSUE TITLE
nz-select nz-option  ngModel  nzValue 不能Reset

### ISSUE THREAD

**Issue by ZhaoRd** [NG-ZORRO/ng-zorro-antd#875]
## What problem does this feature solve?

代码
```
<nz-select name="edition" [nzSize]="'large'" [(ngModel)]="tenant.editionId" (ngModelChange)="onEditionChange($event)">
                <nz-option *ngFor="let edition of editions" [nzLabel]="edition.displayText" [nzValue]="edition.value"></nz-option>
            </nz-select>
```
tenant.editionId 的类型时 number ,
但是 nzValue 只能绑定 string or  object。

这就造成了初始化的时候不能显示已经选中的项

## What does the proposed API look like?

无

**Comment by vthinkxie** [comment]
please provide your demo with stackbiz or plunkr

**Comment by vthinkxie** [comment]
fixed in version 0.7.0
